<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<link href="css/releaseNotes_style.css" rel="stylesheet" type="text/css">
<!-- favicon added -->
<link rel="shortcut icon" href="../html/image/favicon.ico" >

<meta name="description" content="KnetMiner - Mining and prioritizing candidate genes in integrated knowledge networks for plant and animal genomes.">
<meta name="Keywords" content="KnetMiner, tool, Gene, Genome, QTL, Ondex, Plant, Animal, Data Mining, Candidate Gene, Ranking, Prioritization, Bioinformatics" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Fusarium graminearum Release Notes</title>

<script type="text/javascript" src="javascript/releaseNotes/jquery-1.11.2.min.js"></script>
<!-- <script type="text/javascript" src="javascript/dtjava.js"></script> -->
<script type="text/javascript" src="javascript/utils-config.js"></script>  
<script type="text/javascript" src="javascript/releaseNotes/showNetworkStats.js"></script>  

<script src="https://use.fontawesome.com/a90751c39e.js"></script>
<link href="https://fonts.googleapis.com/css?family=Kanit|Play" rel="stylesheet">
</head>

<body id="release-body">

<div id="header">
	<nav class="navbar navbar-default navbar-fixed-top" role="navigation">	  
    	<a href="http://knetminer.rothamsted.ac.uk/" title="KnetMiner Home"><img class="logo-top" src="../html/image/logo-regular.png" alt="Logo" height="50" style="padding-left:10px;"></a>
 	<div id="species_header">Fusarium graminearum</div>

 </nav>  <!-- end navbar -->
</div>	<!-- end header -->
  
<div id="release-content">
<h1 class="release-title">F. graminearum Release Notes</h1>
<p id="network_stats" class="release-title">The final F. graminearum knowledge network contains:<!--<strong>? concepts</strong> and <strong>? relations</strong>.--></p>

<ul>
	<p><strong>Datasets used for integration</strong></p>
	<ul>
	    <li>Gene-Protein [Ensembl]</li>
		<ul>
		    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/gff3/fusarium_graminearum/Fusarium_graminearum.RR1.35.gff3.gz">Fusarium_graminearum.RR1.35.gff3</a></li>
		    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/fasta/fusarium_graminearum/pep/Fusarium_graminearum.RR1.pep.all.fa.gz">Fusarium_graminearum.RR1.pep.all.fa</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=fgraminearum_eg_gene.default.feature_page.ensembl_gene_id|fgraminearum_eg_gene.default.feature_page.ensembl_peptide_id&FILTERS=&VISIBLEPANEL=resultspanel">Mapping File</a></li>
		    <li><a href="http://fungidb.org/common/downloads/release-32/FgraminearumPH-1/txt/FungiDB-32_FgraminearumPH-1_GeneAliases.txt">Old Gene Id</a></li>
		</ul>
	    <li>Protein Domain [Ensembl]</li>
		<ul>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungi_mart&ATTRIBUTES=fgraminearum_eg_gene.default.feature_page.ensembl_peptide_id|fgraminearum_eg_gene.default.feature_page.pfam_pf|fgraminearum_eg_gene.default.feature_page.interpro_id|fgraminearum_eg_gene.default.feature_page.interpro_short_description|fgraminearum_eg_gene.default.feature_page.interpro_description&FILTERS=&VISIBLEPANEL=resultspanel">BioMart InterPro Protein Domains</a></li>
		</ul>
	    <li>Sequence alignments to other Ascomycetes [Decypher, UniprotKB]</li>
		<ul>
		    <li>Protein@F.graminearum-Protein@Ascomycetes::UniProtKB-SP [Smith-Waterman]</li>
		    <li><a href="http://www.uniprot.org/uniprot/?query=reviewed:yes%20taxonomy:4890&compress=yes&format=xml">Ascomycetes XML</a></li>
		</ul>
	    <li>Publications [PubMed]</li>
	    <ul>
		<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=fusarium+graminearum+OR+gibberella+zeae">F.graminearum publications XML</a></li>
		<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=virulence+AND+pathogen*+NOT+virus">XML</a></li>
		<li>filter publications that are unconnected, i.e. not linked via citation or text-mining</li>
	    </ul>
	    <li>Gene-Phenotype [PHI-base]</li>
		<ul>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungi_mart&ATTRIBUTES=fgraminearum_eg_gene.default.feature_page.ensembl_gene_id|fgraminearum_eg_gene.default.feature_page.ensembl_transcript_id|fgraminearum_eg_gene.default.feature_page.phi|fgraminearum_eg_gene.default.feature_page.phibase_host|fgraminearum_eg_gene.default.feature_page.phibase_phenotype|fgraminearum_eg_gene.default.feature_page.phibase_condition&FILTERS=fgraminearum_eg_gene.default.filters.phibase_phenotype.%20increased virulence (hypervirulence),lethal,loss of pathogenicity,mixed outcome,reduced virulence,unaffected pathogenicity%20&VISIBLEPANEL=resultspanel">PHI-base Pathogen Phenotypes</a> from Ensembl BioMart</li>
		</ul>
	    <li>Protein-Phenotype [PHI-base]</li>
		<ul>
		    <li><a href="http://www.phi-base.org/">TAB</a></li>
		    <li>Sequence alignments to PHI-base Proteins [Decypher]</li>
			<ul>
			    <li>Protein@F.graminearum-Protein@PHI-base [Smith-Waterman]</li>
		        </ul>
		</ul>
	    <li>Gene-Phenotype [Text-Mining]</li>
		<ul>
		    <li>Created Gene-Phenotype sentence based co-occurences in PubMed abstracts</li>
		    <li><a href="http://purl.obolibrary.org/obo/apo.owl">apo.owl</a></li>
		    <li><a href="http://purl.obolibrary.org/obo/omp.owl">omp.owl</a></li>
		    <li><a href="http://purl.obolibrary.org/obo/fypo.owl">fypo.owl</a></li>			    
		</ul>
	    <li>Yeast (<i>Saccharomyces cerevisiae</i>)</li>
	        <ul>
		    <li>Gene-Protein [Ensembl]</li>
			<ul>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/gff3/saccharomyces_cerevisiae/Saccharomyces_cerevisiae.R64-1-1.35.gff3.gz">Saccharomyces_cerevisiae.R64-1-1.35.gff3</a></li>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/fasta/saccharomyces_cerevisiae/pep/Saccharomyces_cerevisiae.R64-1-1.pep.all.fa.gz">Saccharomyces_cerevisiae.R64-1-1.pep.all.fa</a></li>
			    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungi_mart&ATTRIBUTES=scerevisiae_eg_gene.default.feature_page.ensembl_gene_id|scerevisiae_eg_gene.default.feature_page.sgd_gene|scerevisiae_eg_gene.default.feature_page.ensembl_transcript_id|scerevisiae_eg_gene.default.feature_page.sgd_transcript&FILTERS=scerevisiae_eg_gene.default.filters.with_sgd_gene.only&VISIBLEPANEL=resultspanel">Ensembl - SGD Mapping</a></li>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/tsv/saccharomyces_cerevisiae/Saccharomyces_cerevisiae.R64-1-1.35.uniprot.tsv.gz">Ensembl - UniProtKB Mapping</a></li>
			</ul>
		    <li>Protein Pathway [YeastCyc, UniProtKB]</li>
		        <ul>
			    <li><a href="ftp://ftp.yeastgenome.org/ptools_data/yeast.tar.gz">YeastCyc Release 15</a></li>
			    <li>biopax-level2.owl</li>
			    <li>translation.tab</li>
			    <li>UniProtKB - BioCyc Mapping</li>
			</ul>
		    <li>Gene-Phenotype [SGD]</li>
		        <ul>
			    <li><a href="ftp://ftp.yeastgenome.org/curation/literature/phenotype_data.tab">phenotype_data.tab</a></li>
			</ul>
		    <li>Gene-Publication [SGD]</li>
			<ul>
			    <li><a href="ftp://ftp.yeastgenome.org/curation/literature/gene_literature.tab">gene_literature.tab</a></li>
			</ul>
		    <li>Protein-Publication [UniProtKB]</li>
			<ul>
			    <li><a href="http://www.uniprot.org/uniprot/?query=proteome:UP000002311&compress=yes&format=xml">Sc Proteome xml</a></li>
			</ul>
		    <li>Protein-Protein Interactions [BioGRID]</li>
			<ul>
			    <li><a href="http://thebiogrid.org/downloads/archives/Release%20Archive/BIOGRID-3.4.146/BIOGRID-ORGANISM-3.4.146.tab2.zip">TAB</a></li>
			    <li>Version 3.4.146</li>
			</ul>
		    <li>Gene-GO [GOA]</li>
			<ul>
			    <li>Include all evidence types</li>
			    <li><a href="ftp://ftp.yeastgenome.org/curation/literature/gene_association.sgd.gz">GAF 2.0</a></li>
			    <li>Ondex GAF parser requires additional mapping file of feature type -> <i>Ondex Concept Class</i></li>
			</ul>
		    <li>Protein Domain [SGD]</li>
			<ul>
			    <li><a href="ftp://ftp.yeastgenome.org/curation/calculated_protein_info/domains/domains.tab">domains.tab</a></li>
			</ul>
		</ul>
	    <li>Aspergillus nidulans</li>
	        <ul>
		    <li>Gene-Protein [Ensembl, AspGD]</li>
		        <ul>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/gff3/aspergillus_nidulans/Aspergillus_nidulans.ASM1142v1.35.gff3.gz">Aspergillus_nidulans.ASM1142v1.35.gff3</a></li>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/fasta/aspergillus_nidulans/pep/Aspergillus_nidulans.ASM1142v1.pep.all.fa.gz">Aspergillus_nidulans.ASM1142v1.pep.all.fa</a></li>
			    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=anidulans_eg_gene.default.feature_page.ensembl_gene_id|anidulans_eg_gene.default.feature_page.ensembl_peptide_id&FILTERS=anidulans_eg_gene.default.filters.biotype.%20protein_coding%20&VISIBLEPANEL=resultspanel">Mapping File</a></li>
			    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=anidulans_eg_gene.default.feature_page.ensembl_gene_id|anidulans_eg_gene.default.feature_page.ensembl_peptide_id|anidulans_eg_gene.default.feature_page.aspgd_gene&FILTERS=&VISIBLEPANEL=resultspanel">Ensembl - AspGD Mapping</a></li>
			    <li><a href="http://www.aspergillusgenome.org/download/External_id_mappings/gp2protein.aspgd.gz">AspGD - UniProtKB Mapping</a></li>
			</ul>
		    <li>Protein-Protein Interactions [BioGRID, Ensembl]</li>
		        <ul>
			    <li><a href="http://thebiogrid.org/downloads/archives/Release%20Archive/BIOGRID-3.4.146/BIOGRID-ORGANISM-3.4.146.tab2.zip">TAB</a></li>
			    <li>Version 3.4.146</li>
			    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungi_mart&ATTRIBUTES=anidulans_eg_gene.default.feature_page.ensembl_peptide_id|anidulans_eg_gene.default.feature_page.wikigene_name&FILTERS=anidulans_eg_gene.default.filters.wikigene_name.%20AN7821.2,AN6197.2,AN6125.2,AN4761.2,AN5105.2,AN5973.2,AN3719.2,AN1052.2,AN3422.2,AN7252.2,AN2269.2,AN1191.2,AN7425.2,AN1459.2,AN4332.2,AN2290.2,AN0807.2,AN1959.2,AN0363.2,AN2142.2,AN2298.2,AN4399.2,AN4869.2,AN5784.2,AN5872.2,AN6547.2,AN2174.2,AN0253.2,AN0440.2,AN0668.2,AN0671.2,AN0976.2,AN1060.2,AN1546.2,AN2141.2,AN2181.2,AN2415.2,AN2769.2,AN3621.2,AN3688.2,AN3907.2,AN4445.2,AN4894.2,AN5498.2,AN6505.2,AN7170.2,AN7608.2,AN9399.2,AN9538.2,AN0161.2,AN1635.2,AN3082.2,AN5124.2,AN0241.2,AN0554.2,AN4258.2,AN1466.2,AN3038.2,AN4622.2,AN4705.2,AN5212.2,AN5532.2,AN6560.2,AN9087.2%20&VISIBLEPANEL=resultspanel">Protein ID Mapping File</a></li>
			</ul>
		    <li>Gene-GO [GOA]</li>
			<ul>
			    <li>Include all evidence types</li>
			    <li><a href="http://www.aspergillusgenome.org/download/go/gene_association.aspgd.gz">GAF 2.0</a></li>
			    <li>Ondex GAF parser requires additional mapping file of feature type -> <i>Ondex Concept Class</i></li>
			</ul>
		    <li>Gene-Phenotype [AspGD]</li>
		        <ul>
			    <li><a href="http://www.aspergillusgenome.org/download/phenotype/A_nidulans_phenotype_data.tab">A_nidulans_phenotype_data.tab</a></li>
			</ul>
		    <li>Protein Domain [AspGD]</li>
		        <ul>
			    <li><a href="http://www.aspergillusgenome.org/download/domains/A_nidulans_FGSC_A4/A_nidulans_FGSC_A4_iprscan.out">A_nidulans_FGSC_A4_iprscan.out</a></li>
			    <li><a href="http://www.aspergillusgenome.org/download/chromosomal_feature_files/A_nidulans_FGSC_A4/A_nidulans_FGSC_A4_current_chromosomal_feature.tab">Protein ID Mapping</a></li>
			</ul>
		    <li>Protein-Publication [UniProtKB]</li>
			<ul>
			    <li><a href="http://www.uniprot.org/uniprot/?query=proteome:UP000000560&compress=yes&format=xml">An Proteome xml</a></li>
			</ul>
		</ul>
	    <li>Neurospora crassa</li>
	        <ul>
		    <li>Gene-Protein [Ensembl]</li>
		        <ul>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/release-35/fungi/fasta/neurospora_crassa/pep/Neurospora_crassa.NC12.pep.all.fa.gz">Neurospora_crassa.NC12.pep.all.fa</a></li>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/release-35/fungi/gff3/neurospora_crassa/Neurospora_crassa.NC12.35.gff3.gz">Neurospora_crassa.NC12.35.gff3</a></li>
			    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=ncrassa_eg_gene.default.feature_page.ensembl_gene_id|ncrassa_eg_gene.default.feature_page.ensembl_transcript_id&FILTERS=&VISIBLEPANEL=resultspanel">Mapping File</a></li>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/tsv/neurospora_crassa/Neurospora_crassa.NC12.35.uniprot.tsv.gz">Ensembl - UniProtKB Mapping</a></li>
			</ul>
		    <li>Protein Domain [Ensembl]</li>
			<ul>
			    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=ncrassa_eg_gene.default.feature_page.ensembl_transcript_id|ncrassa_eg_gene.default.feature_page.pfam|ncrassa_eg_gene.default.feature_page.interpro|ncrassa_eg_gene.default.feature_page.interpro_short_description|ncrassa_eg_gene.default.feature_page.interpro_description&FILTERS=ncrassa_eg_gene.default.filters.with_interpro.only&VISIBLEPANEL=resultspanel">BioMart InterPro Protein Domains</a></li>
			</ul>
		    <li>Protein-Publication [UniProtKB]</li>
			<ul>
			    <li><a href="http://www.uniprot.org/uniprot/?query=proteome:UP000001805&compress=yes&format=xml">Nc Proteome xml</a></li>
			</ul>
		</ul>
	    <li>Zymoseptoria tritici</li>
		<ul>
		    <li>Gene-Protein [Ensembl]</li>
			<ul>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/gff3/zymoseptoria_tritici/Zymoseptoria_tritici.MG2.35.gff3.gz">Zymoseptoria_tritici.MG2.35.gff3</a></li>
			    <li><a href="ftp://ftp.ensemblgenomes.org/pub/fungi/release-35/fasta/zymoseptoria_tritici/pep/Zymoseptoria_tritici.MG2.pep.all.fa.gz">Zymoseptoria_tritici.MG2.pep.all.fa</a></li>
			    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=ztritici_eg_gene.default.feature_page.ensembl_gene_id|ztritici_eg_gene.default.feature_page.ensembl_peptide_id&FILTERS=&VISIBLEPANEL=resultspanel">Mapping File</a></li>
			</ul>
		    <li>Protein Domain [Ensembl]</li>
			<ul>
			    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=ztritici_eg_gene.default.feature_page.ensembl_peptide_id|ztritici_eg_gene.default.feature_page.pfam|ztritici_eg_gene.default.feature_page.interpro|ztritici_eg_gene.default.feature_page.interpro_short_description|ztritici_eg_gene.default.feature_page.interpro_description&FILTERS=ztritici_eg_gene.default.filters.with_interpro.only&VISIBLEPANEL=resultspanel">BioMart InterPro Protein Domains</a></li>
			</ul>
		    <li>Sequence alignments to other Ascomycetes [Decypher, UniprotKB]</li>
			<ul>
			    <li>Protein@Z.tritici-Protein@Ascomycetes::UniProtKB-SP [Smith-Waterman]</li>
			    <li><a href="http://www.uniprot.org/uniprot/?query=reviewed:yes%20taxonomy:4890&compress=yes&format=xml">Ascomycetes XML</a></li>
			</ul>
		    <li>Publications [PubMed]</li>
			<ul>
			    <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=zymoseptoria+tritici+OR+mycospherella+graminicola">Z.tritici publications XML</a></li>
			    <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=virulence+AND+pathogen*+NOT+virus">XML</a></li>
			    <li>filter publications that are unconnected, i.e. not linked via citation or text-mining</li>
			</ul>
		    <li>Gene-Phenotype [PHI-base]</li>
			<ul>
			    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungi_mart&ATTRIBUTES=ztritici_eg_gene.default.feature_page.ensembl_gene_id|ztritici_eg_gene.default.feature_page.ensembl_transcript_id|ztritici_eg_gene.default.feature_page.phi|ztritici_eg_gene.default.feature_page.phibase_host|ztritici_eg_gene.default.feature_page.phibase_phenotype|ztritici_eg_gene.default.feature_page.phibase_condition&FILTERS=ztritici_eg_gene.default.filters.phibase_phenotype.%20chemistry target: resistance to chemical,chemistry target: sensitivity to chemical,effector (plant avirulence determinant),increased virulence (hypervirulence),loss of pathogenicity,reduced virulence,unaffected pathogenicity%20&VISIBLEPANEL=resultspanel">PHI-base Pathogen Phenotypes</a> from Ensembl BioMart</li>
			</ul>
		    <li>Protein-Phenotype [PHI-base]</li>
			<ul>
			    <li><a href="http://www.phi-base.org/">TAB</a></li>
			    <li>Sequence alignments to PHI-base Proteins [Decypher]</li>
				<ul>
				    <li>Protein@Z.tritici-Protein@PHI-base [Smith-Waterman]</li>
				</ul>
			</ul>
		    <li>Gene-Phenotype [Text-Mining]</li>
			<ul>
			    <li>Created Gene-Phenotype sentence based co-occurences in PubMed abstracts</li>
			    <li><a href="http://purl.obolibrary.org/obo/apo.owl">apo.owl</a></li>
			    <li><a href="http://purl.obolibrary.org/obo/omp.owl">omp.owl</a></li>
			    <li><a href="http://purl.obolibrary.org/obo/fypo.owl">fypo.owl</a></li>			    
			</ul>
		</ul>
	    <li>Homology</li>
		<ul>
		    <li>Data was downloaded from Ensembl BioMart</li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=fgraminearum_eg_gene.default.homologs.ensembl_peptide_id|fgraminearum_eg_gene.default.homologs.anidulans_eg_homolog_ensembl_peptide|fgraminearum_eg_gene.default.homologs.anidulans_eg_homolog_orthology_type|fgraminearum_eg_gene.default.homologs.anidulans_eg_homolog_perc_id|fgraminearum_eg_gene.default.homologs.anidulans_eg_homolog_perc_id_r1&FILTERS=fgraminearum_eg_gene.default.filters.with_anidulans_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs F.graminearum - A.nidulans</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=fgraminearum_eg_gene.default.homologs.ensembl_peptide_id|fgraminearum_eg_gene.default.homologs.ncrassa_eg_homolog_ensembl_peptide|fgraminearum_eg_gene.default.homologs.ncrassa_eg_homolog_orthology_type|fgraminearum_eg_gene.default.homologs.ncrassa_eg_homolog_perc_id|fgraminearum_eg_gene.default.homologs.ncrassa_eg_homolog_perc_id_r1&FILTERS=fgraminearum_eg_gene.default.filters.with_ncrassa_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs F.graminearum - N.crassa</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=fgraminearum_eg_gene.default.homologs.ensembl_peptide_id|fgraminearum_eg_gene.default.homologs.scerevisiae_eg_homolog_ensembl_peptide|fgraminearum_eg_gene.default.homologs.scerevisiae_eg_homolog_orthology_type|fgraminearum_eg_gene.default.homologs.scerevisiae_eg_homolog_perc_id|fgraminearum_eg_gene.default.homologs.scerevisiae_eg_homolog_perc_id_r1&FILTERS=fgraminearum_eg_gene.default.filters.with_scerevisiae_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs F.graminearum - Yeast</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=ztritici_eg_gene.default.homologs.ensembl_peptide_id|ztritici_eg_gene.default.homologs.fgraminearum_eg_homolog_ensembl_peptide|ztritici_eg_gene.default.homologs.fgraminearum_eg_homolog_orthology_type|ztritici_eg_gene.default.homologs.fgraminearum_eg_homolog_perc_id|ztritici_eg_gene.default.homologs.fgraminearum_eg_homolog_perc_id_r1&FILTERS=ztritici_eg_gene.default.filters.with_fgraminearum_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs F.graminearum - Z.tritici</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=ztritici_eg_gene.default.homologs.ensembl_peptide_id|ztritici_eg_gene.default.homologs.anidulans_eg_homolog_ensembl_peptide|ztritici_eg_gene.default.homologs.anidulans_eg_homolog_orthology_type|ztritici_eg_gene.default.homologs.anidulans_eg_homolog_perc_id|ztritici_eg_gene.default.homologs.anidulans_eg_homolog_perc_id_r1&FILTERS=ztritici_eg_gene.default.filters.with_anidulans_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs Z.tritici - A.nidulans</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=ztritici_eg_gene.default.homologs.ensembl_peptide_id|ztritici_eg_gene.default.homologs.ncrassa_eg_homolog_ensembl_peptide|ztritici_eg_gene.default.homologs.ncrassa_eg_homolog_orthology_type|ztritici_eg_gene.default.homologs.ncrassa_eg_homolog_perc_id|ztritici_eg_gene.default.homologs.ncrassa_eg_homolog_perc_id_r1&FILTERS=ztritici_eg_gene.default.filters.with_ncrassa_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs Z.tritici - N.crassa</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=ztritici_eg_gene.default.homologs.ensembl_peptide_id|ztritici_eg_gene.default.homologs.scerevisiae_eg_homolog_ensembl_peptide|ztritici_eg_gene.default.homologs.scerevisiae_eg_homolog_orthology_type|ztritici_eg_gene.default.homologs.scerevisiae_eg_homolog_perc_id|ztritici_eg_gene.default.homologs.scerevisiae_eg_homolog_perc_id_r1&FILTERS=ztritici_eg_gene.default.filters.with_scerevisiae_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs Z.tritici - Yeast</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=anidulans_eg_gene.default.homologs.ensembl_peptide_id|anidulans_eg_gene.default.homologs.ncrassa_eg_homolog_ensembl_peptide|anidulans_eg_gene.default.homologs.ncrassa_eg_homolog_orthology_type|anidulans_eg_gene.default.homologs.ncrassa_eg_homolog_perc_id|anidulans_eg_gene.default.homologs.ncrassa_eg_homolog_perc_id_r1&FILTERS=anidulans_eg_gene.default.filters.with_ncrassa_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs A.nidulans - N.crassa</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=anidulans_eg_gene.default.homologs.ensembl_peptide_id|anidulans_eg_gene.default.homologs.scerevisiae_eg_homolog_ensembl_peptide|anidulans_eg_gene.default.homologs.scerevisiae_eg_homolog_orthology_type|anidulans_eg_gene.default.homologs.scerevisiae_eg_homolog_perc_id|anidulans_eg_gene.default.homologs.scerevisiae_eg_homolog_perc_id_r1&FILTERS=anidulans_eg_gene.default.filters.with_scerevisiae_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs A.nidulans - Yeast</a></li>
		    <li><a href="http://fungi.ensembl.org/biomart/martview?VIRTUALSCHEMANAME=fungal_mart&ATTRIBUTES=ncrassa_eg_gene.default.homologs.ensembl_transcript_id|ncrassa_eg_gene.default.homologs.scerevisiae_eg_homolog_ensembl_peptide|ncrassa_eg_gene.default.homologs.scerevisiae_eg_homolog_orthology_type|ncrassa_eg_gene.default.homologs.scerevisiae_eg_homolog_perc_id|ncrassa_eg_gene.default.homologs.scerevisiae_eg_homolog_perc_id_r1&FILTERS=ncrassa_eg_gene.default.filters.with_scerevisiae_eg_homolog.only&VISIBLEPANEL=resultspanel">Homologs N.crassa - Yeast</a></li>
		</ul>
	    <li>Ontologies</li>
		<ul>
		    <li><a href="ftp://ftp.geneontology.org/pub/go/ontology/go-basic.obo">go-basic.obo</a></li>
		    <li><a href="ftp://ftp.geneontology.org/pub/go/external2go/ec2go">ec2go.txt</a></li>
		</ul>
	    
</ul>

</div>        <!-- end div release-content  -->


<div class="contact-footer"> 
<ul>                       		
      <li style="float: left; width: 23%;"><a target="_blank" title="Rothamsted Research" href="http://www.rothamsted.ac.uk" class="logos"><img src="../html/image/rothamsted_logo.png" width="63.6" height="85.5"/></a>  
       <a target="_blank" title="BBSRC" href="http://www.bbsrc.ac.uk" class="logos"><img class="bbsrc-logo" src="../html/image/bbsrc_logo.png" width="194" height="78"/></a></li>
      <li style="float: left; width: 50%; margin: 0px 2%;"><img class="knetminer-logo" src="../html/image/logo-regular.png" alt="Logo" with="201.5" height="66.3">
        <p id="text-logo">Supported by BBSRC SABR and TRDF awards BB/F006039/1 and BB/I023860/1</p>  </li>      
      <li style="float: left; width: 20%;"><a class="contact-button" title="Contact Page" href="Contact.html"><h4 class="contact-button"><i class="fa fa-envelope-o" aria-hidden="true"></i>&nbsp;&nbsp;Contact and find us</h4></a>
        <p style="font-style:normal;">keywan.hassani-pak@rothamsted.ac.uk </p>
        <div class="twitter-buttons">
        <a href="https://twitter.com/KnetMiner" class="twitter-follow-button" data-show-count="false">Follow @KnetMiner</a><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
           </div>
           <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>   
 	</li>
 	</div>


</body>
</html>
