<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<link href="css/releaseNotes_style.css" rel="stylesheet" type="text/css">
<!-- favicon added -->
<link rel="shortcut icon" href="../html/image/favicon.ico" >

<meta name="description" content="KnetMiner - Mining and prioritizing candidate genes in integrated knowledge networks for plant and animal genomes.">
<meta name="Keywords" content="KnetMiner, tool, Gene, Genome, QTL, Ondex, Plant, Animal, Data Mining, Candidate Gene, Ranking, Prioritization, Bioinformatics" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Arabidopsis Release Notes</title>

<script type="text/javascript" src="javascript/releaseNotes/jquery-1.11.2.min.js"></script>
<!-- <script type="text/javascript" src="javascript/dtjava.js"></script> -->
<script type="text/javascript" src="javascript/utils-config.js"></script>  
<script type="text/javascript" src="javascript/releaseNotes/showNetworkStats.js"></script>  

<script src="https://use.fontawesome.com/a90751c39e.js"></script>
<link href="https://fonts.googleapis.com/css?family=Kanit|Play" rel="stylesheet">
</head>

<body id="release-body">

<div id="header">
	<nav class="navbar navbar-default navbar-fixed-top" role="navigation">	  
    	<a href="http://knetminer.rothamsted.ac.uk/" title="KnetMiner Home"><img class="logo-top" src="../html/image/logo-regular.png" alt="Logo" height="50" style="padding-left:10px;"></a>
 	<div id="species_header">Arabidopsis thaliana</div>

 </nav>  <!-- end navbar -->
</div>	<!-- end header -->
  
<div id="release-content">
<h1 class="release-title">Arabidopsis Release Notes</h1>
<p id="network_stats" class="release-title">The final Arabidopsis thaliana information network contains:<!--<strong>219,033 concepts</strong> and <strong>817,619 relations</strong>.--></p>

<ul>

	<p><strong>Datasets used for integration</strong></p>
	<ul>
		<li>Gene- Protein [Ensembl]</li>
		<ul>
		    <li><a href="ftp://ftp.ensemblgenomes.org/pub/current/plants/gff3/arabidopsis_thaliana/Arabidopsis_thaliana.TAIR10.36.gff3.gz">Arabidopsis_thaliana.TAIR10.36.gff3</a></li>
                    <li><a href="ftp://ftp.ensemblgenomes.org/pub/current/plants/fasta/arabidopsis_thaliana/pep/Arabidopsis_thaliana.TAIR10.pep.all.fa.gz">Arabidopsis_thaliana.TAIR10.pep.all.fa</a></li>
		    <li><a href="ftp://ftp.ensemblgenomes.org/pub/plants/current/tsv/arabidopsis_thaliana/Arabidopsis_thaliana.TAIR10.36.uniprot.tsv.gz">Ensembl - UniprotKB Mapping</a></li>
		</ul>
    	<li>Gene-SNP-Phenotype [Ensembl]</li>
    	<ul>
            <li>Arabidopsis GWAS data for 92 phenotypes.</li>
            <li>Phenotypes are grouped into the four categories: flowering, developmental, defense and ionomics.</li>
            <li>For more information see <a href="http://www.nature.com/nature/journal/v465/n7298/abs/nature08800.html">Atwell et al, 2010, Nature</a> and <a href="http://www.nature.com/nature/journal/v465/n7298/extref/nature08800-s1.pdf">supplementary information</a></li>
            <li>Data was downloaded from <a href="Protein@Rice-Protein@Plants::UniProtKB-SP [Smith-Waterman]">Ensembl Biomart</a></li>
            <li>SNP-Phenotype relations (27,466 relations) of significant SNPs (p-value<0.0001) linked to 92 phenotypes; on average 300 SNPs per phenotype.</li>
            <li>SNP-Gene relations are based on genes in close proximity to SNPs <1000bp (13,331 relations)</li>
            <li>#SNP=11,996 | #Gene=8,241 | #Phenotype=92</li>
            <li>To see it live, go to <a href="http://knetminer.rothamsted.ac.uk/Arabidopsis_thaliana/">KnetMiner Arabidopsis</a> and search for <i>dormancy</i>, then switch to Map View.</li>
        </ul>
    	<li>Gene-GO [GOA]</li>
    	<ul>
            <li>Include all evidence types</li>
            <li><a href="http://www.geneontology.org/gene-associations/gene_association.tair.gz">GAF 2.0</a></li>
            <li>Ondex GAF parser requires additional mapping file of feature type -> <i>Ondex Concept Class</i></li>
        </ul>
    	<li>Gene-Gene Interactions [BioGRID]</li>
    	<ul>
            <li><a href="http://thebiogrid.org/downloads/archives/Release%20Archive/BIOGRID-3.4.148/BIOGRID-ORGANISM-3.4.151.tab2.zip">TAB</a></li>
            <li>Version 3.4.151</li>
            <li>Concepts: 9212 [Gene] | 2359 [Publication]</li>
            <li>Relations: 303 [genetic] | 37083 [physical] | 19295 [pub_in]</li>
	    <li>filter out High Throughput interactions for gene AT5G03240</li>
        </ul>
    	<li>Gene-Phenotype [TAIR]</li>
    	<ul>
            <li><a href="ftp://ftp.arabidopsis.org/home/tair/User_Requests/Locus_Germplasm_Phenotype_20130122">TAB</a></li>
        </ul>
    	<li>Publications [PubMed]</li>
    	<ul>
            <li><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=arabidopsis+thaliana">XML</a></li>
            <li>57,600 PubMed abstracts</li>
            <li>filter publications that are unconnected, i.e. not linked via citation or text-mining</li>
        </ul>
    	<li>Gene-Publication [GOA, TAIR]</li>
    	<ul>
            <li><a href="http://www.geneontology.org/gene-associations/gene_association.tair.gz">GAF 2.0</a></li>
            <li><a href="ftp://ftp.arabidopsis.org/home/tair/User_Requests/Locus_Published_20140331.txt.gz">TAB</a></li>
        </ul>
    	<li>Protein-Publication [UniProtKB, TAIR]</li>
    	<ul>
            <li><a href="http://www.uniprot.org/uniprot/?query=proteome:UP000006548&compress=yes&format=xml">At proteome XML</a></li>
            <li><a href="ftp://ftp.arabidopsis.org/home/tair/User_Requests/Locus_Published_20140331.txt.gz">TAB</a></li>
        </ul>
    	<li>Protein-Pathway [AraCyc]</li>
    	<ul>
            <li>AraCyc Release 15</li>
            <li>biopax-level2.owl</li>
            <li>Translation.tab</li>
        </ul>
	<li>Protein Domain [Ensembl]</li>
	<ul>
	    <li>InterPro Protein Domains from <a href="http://plants.ensembl.org/biomart/martview">Ensembl BioMart</a></li>
	</ul>
    	<li>Gene-TO [Text Mining]</li>
    	<ul>
            <li>Created 27k Gene-TO sentence based co-occurrences in PubMed abstracts</li>
        </ul>
    	<li>Sequence alignments to other Plants</li>
    	<ul>
            <li>Protein@Arabidopsis-Protein@Plants::UniProtKB-SP [Smith-Waterman]:
                <ul>
		    <li>Concepts: 27112 [Protein]</li>
                    <li>Relations: 102202 [h_s_s]</li>
                </ul>
            </li>
            <li>Protein-GO [UniProtKB]</li>
            <li>Protein-Publication [UniProtKB]</li>
	    <li><a href="http://www.uniprot.org/uniprot/?query=taxonomy%3A33090+NOT+taxonomy%3A3702+AND+reviewed%3Ayes&compress=yes&format=xml">UniprotKB-SP Plants XML</a></li>
        </ul>
    	<li>Orthologs in Yeast</li>
    	<ul>
            <li>Protein@Arabidopsis-Protein@Yeast [OMA]</li>
            <li>Protein-GO [UniProtKB]</li>
            <li>Protein-Publication [UniProtKB]</li>
	    <li>SGD-UniprotKB [Ensembl] <a href="ftp://ftp.ensemblgenomes.org/pub/fungi/current/tsv/saccharomyces_cerevisiae/Saccharomyces_cerevisiae.R64-1-1.36.uniprot.tsv.gz">TAB</a></li>
            <li>Protein-Protein Interaction [BioGRID]:
                <ul>
		    <li><a href="http://thebiogrid.org/downloads/archives/Release%20Archive/BIOGRID-3.4.148/BIOGRID-ORGANISM-3.4.151.tab2.zip">TAB</a></li>
		    <li>Concepts: 5344 [Protein] | 13954 [Publication]</li>
                    <li>Relations: 31041 [genetic] | 27773 [physical] | 72752 [pub_in]</li>
		    <li>filter out High Throughput interactions</li>
                </ul>
            </li>
        </ul>

</ul>

</div>        <!-- end div release-content  -->


<div class="contact-footer"> 
<ul>                       		
      <li style="float: left; width: 23%;"><a target="_blank" title="Rothamsted Research" href="http://www.rothamsted.ac.uk" class="logos"><img src="../html/image/rothamsted_logo.png" width="63.6" height="85.5"/></a>  
       <a target="_blank" title="BBSRC" href="http://www.bbsrc.ac.uk" class="logos"><img class="bbsrc-logo" src="../html/image/bbsrc_logo.png" width="194" height="78"/></a></li>
      <li style="float: left; width: 50%; margin: 0px 2%;"><img class="knetminer-logo" src="../html/image/logo-regular.png" alt="Logo" with="201.5" height="66.3">
        <p id="text-logo">Supported by BBSRC SABR and TRDF awards BB/F006039/1 and BB/I023860/1</p>  </li>      
      <li style="float: left; width: 20%;"><a class="contact-button" title="Contact Page" href="Contact.html"><h4 class="contact-button"><i class="fa fa-envelope-o" aria-hidden="true"></i>&nbsp;&nbsp;Contact and find us</h4></a>
        <p style="font-style:normal;">keywan.hassani-pak@rothamsted.ac.uk </p>
        <div class="twitter-buttons">
        <a href="https://twitter.com/KnetMiner" class="twitter-follow-button" data-show-count="false">Follow @KnetMiner</a><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
           </div>
           <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>   
 	</li>
 	</div>


</body>
</html>
